spotlight-client/src/contentApi/sources/us_me.ts

```diff
@@ -0,0 +1,455 @@
+        },
+        {
+          metricTypeId: "PrisonRecidivismRateHistorical",
+          title: "How many people end up back in prison?",
+          body:
+            "After release from prison, a significant proportion of formerly incarcerated individuals end up back in prison. This is typically termed “recidivism.” This chart shows recidivism as reincarceration; that is, the proportion of individuals who are incarcerated again at some point after their release.<p><strong>Note:</strong> Race or Ethnicity, Gender, and Age Group views are disabled unless a single Cohort is selected.</p>",
+        },
+        {
+          metricTypeId: "PrisonRecidivismRateSingleFollowupHistorical",
+          title: "How has the recidivism rate changed over time?",
+          body:
+            "We can also observe the recidivism rate over time for a given number of years after original release.",
+        },
+      ],
+    },
+    Probation: {
+      title: "Probation",
+      previewTitle: "Revocations from Probation, by Type",
+      introduction:
+        "Probation refers to adults whom the courts place on supervision in the community in lieu of or in addition to incarceration. In Maine, probation is managed by the Department of Correction (DOC).",
+      preview: "ProbationRevocationsAggregate",
+      sections: [
+        {
+          title: "Who is on probation?",
+          body:
+            "Judges may sentence people to a period of probation for a Class A misdemeanor crime or greater. Probation can be either a suspended sentence in which the judge has decided on a carceral sentence but has declined to carry it out unless the defendant does not successfully complete a period of probation supervision, or a deferred sentence, in which the defendant has an opportunity for the crime to be recorded as “dismissed” on the criminal record.",
+          metricTypeId: "ProbationPopulationCurrent",
+        },
+        {
+          title: "How has the probation population changed over time?",
+          body:
+            "Broadly speaking, increased activity in earlier parts of the criminal justice system (such as arrests and sentencing) will result in increases in the probation population. Changes in probation sentence lengths, etc. may also contribute to the rise and fall of this number.",
+          metricTypeId: "ProbationPopulationHistorical",
+        },
+        {
+          title: "Why do revocations happen?",
+          body:
+            "Revocations happen when a person on probation violates a condition of their supervision or commits a new crime. In Maine, probation revocations fall into one of three categories: technical violation, new offense, and absconsion.",
+          metricTypeId: "ProbationRevocationsAggregate",
+        },
+      ],
+    },
+  },
+  // TODO(#540): Update localities
```

Comment:
Is this the right TODO for this?

Reply:
Hmm I see this in both us_id.ts and us_tn.ts, but it points to https://github.com/Recidiviz/public-dashboard/issues/540. @colincadams is this the right TODO?